koji-prometheus-exporter.py

```diff
@@ -254,6 +259,13 @@ def scrape():
     for value, labels in koji_tasks_total(error_tasks):
         koji_task_errors_total_family.add_metric(labels, value)
 
+    koji_task_completed_total_family = CounterMetricFamily(
+        'koji_task_completed_total', 'Count of all koji task completed', labels=TASK_LABELS
```

Comment:
Maybe this should be `koji_task_completions_total`.  I think there's supposed to be a way you read these metric names in english.  From the right to the left: the thing is a total. A total of what? completions. What kind of completions? tasks, specifically koji tasks.

That's why `koji_task_errors_total` is the way it is - the a total count of errors of koji tasks.

I think `koji_tasks_completed_total` would also work and maybe feels more natural, but `koji_task_completions_total` is symmetric with `koji_task_errors_total`.

I dunno - WDYT?

Reply:
FWIW, I'm leaning on https://prometheus.io/docs/practices/naming/ here.